### DISCUSSION TITLE
[QUESTION] Moving branches bases from parallel to in-series (with respect to main/master)

### DISCUSSION THREAD

**Discussion by Cooltank10** [community/community#68009] [New to GitHub]
Hi, new to Github and Git, and I was wondering how to change my branch tree from this:

- Active - README
  - Updates - README
  - Experimental - README

to this:

- Active - README
  - Updates - README
    - Experimental - README

(this is all I have in my repository so far)

I already have latest version of git installed. Also not sure how I can use my Desktop VSCode to link with Github, if even possible. I also am new to using the terminal for pretty much anything.
Also anything that could be a good source for a beginner at git/Github would be greatly appreciated!

**Comment by seyLu** [comment]
#### Hi @Cooltank10, 

I recommend reading up on `Github Flow`. But before that, familiarize with `Git` first.

If you want video courses on Git:
- [[1] Youtube: FCC Git and Github Crash Course](https://www.youtube.com/watch?v=RGOj5yH7evk)
- [[2] Youtube: FCC Git for Professionals](https://www.youtube.com/watch?v=Uszj_k0DGsg)

And optionally you can also take:
- [[3] Youtube: FCC Advanced Git](https://www.youtube.com/watch?v=qsTthZi23VE)

And for Github Flow:
- [[4] Youtube: A typical Github workflow](https://www.youtube.com/watch?v=02aQhH5cNBg)

Or if you prefer learning through games:
- [ohmygit](https://ohmygit.org/)
- [learngitbranching](https://learngitbranching.js.org/)

Or if you prefer reading, you can read more on:
- [Git - The simple guide](https://rogerdudler.github.io/git-guide/)
- [Intro to Git and Github](https://product.hubspot.com/blog/git-and-github-tutorial-for-beginners)
- [Git docs](https://git-scm.com/doc)
- [Github docs: Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow)
